playground/mocks/data/idp/idx/device-assurance-grace-period-due-by-date.json

```diff
@@ -31,11 +31,11 @@
       "type": "array",
       "value": [
           {
-            "message": "Your device doesn't meet the security requirements. Fix the issue by 08/01/2024 to prevent lockout.",
+            "message": "Your device doesn't meet the security requirements. Fix the issue by 1725667200000 to prevent lockout.",
```

Comment:
Does server actually send such `message`?

Reply:
Yes. We don't actually use the `message` field at all on the client side since we have the matching i18n key in our SIW bundle, but we needed to send the strings so that we can pass along the parameters. In hindsight, we could have maybe explored passing these expiry values within the `remediation` section of the IDX response, but I don't think I've seen too much precedent for doing that when the value isn't also directly tied to a form control.

I just checked and the response actually renders the timestamp parameter with commas like: 
_"Your device doesn't meet the security requirements. Fix the issue by 1,726,790,400,000 to prevent lockout."_

Reply:
Overall, it's an awkward pattern but the server-side `message` field should never actually get rendered anywhere.